specification/riskiq/Easm/main.tsp

```diff
@@ -0,0 +1,42 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@typespec/versioning";
+import "./routes.tsp";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using TypeSpec.Versioning;
+
+@useAuth(OAuth2Auth<[ImplicitFlow]>)
```

Comment:
I think you need to define this like in the example here: https://github.com/Azure/typespec-azure/blob/907bd205e66943dc3d5e89947d8de9d826f0dbf4/packages/samples/specs/data-plane/languageqna/Interface/main.tsp#L7-L13

@markcowl is the definition I linked to in the sample the correct way to define an implicit auth flow?